permission_handler_android/android/src/main/java/com/baseflow/permissionhandler/PermissionUtils.java

```diff
@@ -292,19 +300,42 @@ static List<String> getManifestNames(Context context, @PermissionConstants.Permi
                 break;
             }
             case PermissionConstants.PERMISSION_GROUP_NOTIFICATION:
-                // The POST_NOTIFICATIONS permission is introduced in Android 13, meaning we should
-                // not handle permissions on pre Android 13 devices.
+                // The POST_NOTIFICATIONS permission is introduced in Android T, meaning we should
+                // not handle permissions on pre Android T devices.
                 if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU && hasPermissionInManifest(context, permissionNames, Manifest.permission.POST_NOTIFICATIONS ))
                     permissionNames.add(Manifest.permission.POST_NOTIFICATIONS);
                 break;
             case PermissionConstants.PERMISSION_GROUP_NEARBY_WIFI_DEVICES:
-            // The NEARBY_WIFI_DEVICES permission is introduced in Android 13, meaning we should
-            // not handle permissions on pre Android 13 devices.
+            // The NEARBY_WIFI_DEVICES permission is introduced in Android T, meaning we should
+            // not handle permissions on pre Android T devices.
                 if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU && hasPermissionInManifest(context, permissionNames, Manifest.permission.NEARBY_WIFI_DEVICES ))
                     permissionNames.add(Manifest.permission.NEARBY_WIFI_DEVICES);
                 break;
-            case PermissionConstants.PERMISSION_GROUP_MEDIA_LIBRARY:
             case PermissionConstants.PERMISSION_GROUP_PHOTOS:
+                // The READ_MEDIA_IMAGES permission is introduced in Android T, meaning we should
+                // not handle permissions on pre Android T devices.   
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU && hasPermissionInManifest(context, permissionNames, Manifest.permission.READ_MEDIA_IMAGES ))
+                    permissionNames.add(Manifest.permission.READ_MEDIA_IMAGES);
+                break;
+                case PermissionConstants.PERMISSION_GROUP_VIDEOS:
```

Comment:
The alignment of the `case` statement is incorrect, initially I completely overlooked it and wanted to make a remark that video permissions where missing.